client/reader/discover/components/tags-navigation/index.tsx

```diff
@@ -56,15 +58,17 @@ export const useRecommendedTags = (): Tag[] => {
 
 const DiscoverTagsNavigation = ( { selectedTag, width, onTagSelect }: Props ) => {
 	const recommendedTags = useRecommendedTags();
+	const dispatch = useDispatch();
 
-	const recordTabClick = () => {
-		recordAction( 'click_discover_tab' );
-		recordGaEvent( 'Clicked Discover Tab' );
```

Comment:
you can see we used the same strings at the level above in the navigation/v2/index. I propose we change it to "tags" here.

"Tabs" was previously (v1) used at the top level of discover navigation (when that represented tags), and now i think it still makes sense to use "tabs" for the top level discover navigation that are now more general tabs. Now that tags are more nested, lets just update the string to be contextual/non-redundant.